openprescribing/measure_definitions/opioidome.json

```diff
@@ -1,42 +1,47 @@
 {
-	"name": "THIS MEASURE IS CURRENTLY SUSPENDED: Prescribing of opioids (total oral morphine equivalence)",
-	"title": [
-		""
-	],
-	"description": [
-		"This measure is currently suspended pending a review of the changes of opioid equivalence by the Faculty of Pain Medicines Opioids Aware group."
-	],
-	"numerator_short": "",
-	"denominator_short": "",
-	"why_it_matters": [
-		""
-	],
-	"tags": [],
-	"url": null,
-	"is_percentage": false,
-	"is_cost_based": false,
-	"low_is_good": null,
-	"include_in_alerts": false,
-	"numerator_type": "custom",
-	"numerator_columns": [
-		"SUM(total_ome) AS numerator"
-	],
-	"numerator_from": "{measures}.opioid_total_ome ",
-	"numerator_where": [
-		"1 = 1"
-	],
-	"numerator_bnf_codes_query": [
-		"SELECT DISTINCT presentation.bnf_code",
-		"FROM {hscic}.presentation",
-		"JOIN ebmdatalab.richard.opioid_measure",
-		"ON CONCAT(",
-		"    SUBSTR(presentation.bnf_code,0,9),",
-		"    'AA',",
-		"    SUBSTR(presentation.bnf_code,-2,2)",
-		"  ) = CONCAT(",
-		"    SUBSTR(opioid_measure.bnf_code,0,11),",
-		"    SUBSTR(opioid_measure.bnf_code,-2,2)",
-		"  )"
-	],
-	"denominator_type": "list_size"
-} 
+  "name": "Prescribing of opioids (total oral morphine equivalence)",
+  "title": [
+    "Total opioid prescribing (as oral morphine equivalence) per 1000 patients"
+  ],
+  "description": [
+    "Total opioid prescribing (as oral morphine equivalence) per 1000 patients"
+  ],
+  "numerator_short": "Total opioids prescribed (in OME mg)",
+  "denominator_short": "1000 patients",
+  "why_it_matters": [
+    "*TO BE AMENDED* The <a href='https://www.rcoa.ac.uk/faculty-of-pain-medicine/opioids-aware'> Opioids Aware</a> project seeks to ",
```

Comment:
Just to highlight that the TO BE AMENDED is still there in this text. Is this intentional?

Reply:
yes, i need to make sure it's right before I publish!